data/src/main/kotlin/com/vultisig/wallet/data/api/SolanaApi.kt

```diff
@@ -133,6 +133,8 @@ internal class SolanaApiImp @Inject constructor(
                 method = "sendTransaction",
                 params = buildJsonArray {
                     add(tx)
+                    addJsonObject {
+                        put("maxRetries",10) }
```

Comment:
Why `blockhash not found` happen?  does add `maxRetries` fix the issue?

Reply:
On Solana, each transaction includes a `blockhash` (a reference to a recent block) to ensure that the transaction is not replayed in the future. This `blockhash` has a limited lifespan. If your transaction takes too long to be processed or broadcasted, the `blockhash` could expire, resulting in the "`blockhash not found`" error.

By adding `"maxRetries": 10`, we are instructing the RPC node to retry sending the transaction up to 10 times before giving up. This can help with cases where the `blockhash` expires before the transaction is successfully broadcasted. During these retries, the node may regenerate a valid `blockhash` and re-attempt sending the transaction, thus preventing the `"blockhash not found"` error.

Relying on public RPC nodes can lead to higher network congestion and latency, which increases the chances of encountering `blockhash not found `errors due to delays in processing transactions.